java/src/main/native/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 # =============================================================================
-# Copyright (c) 2019-2022, NVIDIA CORPORATION.
+# Copyright (c) 2019-2023, NVIDIA CORPORATION.
```

Comment:
Copyright update with no other changes to the file?

Reply:
Hmm must have been a pre-commit hook, looks like there were a few. Let me revert.

Reply:
Removed, thanks.